packages/website/lib/api.js

```diff
@@ -3,7 +3,7 @@ import constants from './constants'
 
 export const API = constants.API
 
-const LIFESPAN = 900
+const LIFESPAN = 60 * 60 * 2 // 2 hours
```

Comment:
Is there a way to auto renew the token automatically if the user is still on the page?

This will do for now but won't fix for really big uploads.

Reply:
not really, we gonna need to discuss this further theres no good option right now